stake-o-matic/src/main.rs

```diff
@@ -1134,8 +1134,11 @@ fn main() -> Result<(), Box<dyn error::Error>> {
     let (quality_block_producers, poor_block_producers, cluster_average_skip_rate) =
         classify_block_producers(&rpc_client, &config, last_epoch)?;
 
-    let too_many_poor_block_producers = poor_block_producers.len()
-        > quality_block_producers.len() * config.max_poor_block_producer_percentage / 100;
+    let too_many_poor_block_producers = if quality_block_producers.is_empty() {
+        true
+    } else {
+        poor_block_producers.len() > quality_block_producers.len() * config.max_poor_block_producer_percentage / 100
```

Comment:
When quality_block_producers.is_empty(), we previously had:
`poor_block_producers.len() > 0 * config.max_poor_block_producer_percentage / 100`

So no quality block producers means that `too_many_poor_block_producers` is false and we destake the world!